test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandler.ValidateTokenAsyncTests.Lifetime.Extensibility.cs

```diff
@@ -19,7 +19,7 @@ public async Task ValidateTokenAsync_Lifetime_Extensibility(ValidateTokenAsyncLi
         {
             var context = TestUtilities.WriteHeader($"{this}.{nameof(ValidateTokenAsync_Lifetime_Extensibility)}", theoryData);
 
-            string jwtString = CreateToken(theoryData.IssuedAt, theoryData.NotBefore, theoryData.Expires);
+            string jwtString = CreateTokenWithLifetime(theoryData.IssuedAt, theoryData.NotBefore, theoryData.Expires);
```

Comment:
May I ask why we removed CreateToken and replaced with this one instead? I'm trying to think about keeping our test classes localized and as simple as possible. 🤔

Reply:
This may be a bit tricky when looking at one PR in isolation, but `JsonWebTokenHandler.ValidateTokenAsyncTests` is a class spread out over many partials in different files.
Because the private methods are still part of the overall class, the names of the multiple `CreateToken` methods we have across files collide. The one that you see removed in this file is because I accidentally created two methods that did the exact same thing in the Lifetime comparison and Lifetime extensibility tests, so I removed one of the copies and called the other, for simplicity.

Reply:
great question @kellyyangsong, thanks for the response @iNinja

Reply:
@iNinja thank you for taking the time to explain!